protocols/3db190/README.md

```diff
@@ -1,10 +1,10 @@
-# PCR/qPCR Prep
+# Lyra Direct SARS-CoV Assay Sample Prep
 
 ### Author
 [Opentrons](https://opentrons.com/)
 
 ## Categories
-* Featured
+* PCR
```

Comment:
This should still be listed as Featured (since it's top 10 popular protocol in 2021)